mindmeld/active_learning/classifiers.py

```diff
@@ -168,9 +172,70 @@ def _validate_class_level_statistic(class_level_statistic):
             )
         return class_level_statistic
 
+    @staticmethod
+    def _get_entity_probs(
+        classifier: Classifier,
+        queries: ProcessedQueryList,
+        tag_to_id: Dict,
+    ):
+        """Get the probability distribution for a query across domains or intents.
+
+        Args:
+            classifier (MindMeld Classifer): Domain or Intent Classifier
+            queries (ProcessedQueryList): List of MindMeld queries
+            tag_to_id (Dict): Dictionary mapping domain or intent names to vector index
+                positions.
+
+        Returns:
+            prob_vector (List[List]]): Probability distribution vectors for given queries.
+        """
+        queries_prob_vectors = []
+        if queries:
+            classifier_eval = classifier.evaluate(queries=queries, active_learning=True)
+
+            domain = classifier.domain
+            intent = classifier.intent
+            default_prob = 1.0
+            default_tag = "O|"
+            default_key = f"{domain}.{intent}.{default_tag}"
+            default_idx = tag_to_id[f"{default_key}"]
+
+            if not classifier_eval:
```

Comment:
We can either add a comment here explaining what "not classifier_eval" means and why it happens or include this descriptive variable name (or both).

```
intent_lacks_entities = not classifier:
if intent_lacks_entities:
      # Code here
```